lib/sass/calcite-web/type/_faces.scss

```diff
@@ -15,31 +16,37 @@
 }
 
 @mixin avenir-regular() {
+  font-family: 'Avenir Next W06 Regular','Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
   font-weight: 400; font-style: normal;
   b, strong {font-weight: 500;}
 }
 
 @mixin avenir-italic() {
+  font-family: 'AvenirNextLTW02-Italic','Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
```

Comment:
This font stack should probably have Avenir and Avenir Next in it somewhere so that if the web fonts don't work or timeout for whatever reason the user's system Avenir takes over. All mac users will have Avenir Next + Avenir installed on their system.